src/service.rs

```diff
@@ -255,12 +258,12 @@ impl Service {
                             let _ = cmd_sender.send(Closure::new(move |state: &mut State| {
                                 let _ = state.handle_connect(token, h, t);
                             }));
-                        },
+                        }
                         Err(e) => {
                             let _ = cmd_sender.send(Closure::new(move |state: &mut State| {
                                 let _ = state.event_sender.send(Event::OnConnect(Err(e), token));
                             }));
-                        },
+                        }
```

Comment:
removing comma here doesn't seem suspicious, but the comma is a nice marker for the end of case and works better at merging patches (because if patch add a new case at the end, it doesn't need to modify previous line and I believe handing multiple of these patches is easier to automate if they don't try to change previous line so I believe git has a better time if we keep the ending comma)